docker/scripts/wait_for_db.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
```

Comment:
https://www.digitalocean.com/community/tutorials/containerizing-a-node-js-application-for-development-with-docker-compose